himl/config_generator.py

```diff
@@ -197,11 +197,11 @@ def generate_hierarchy(self):
         """
         hierarchy = []
         full_path = pathlib2.Path(self.path)
+        os.chdir(self.cwd)
         for path in full_path.parts:
             os.chdir(path)
             new_path = os.path.relpath(os.getcwd(), self.cwd)
             hierarchy.append(self.get_yaml_from_path(new_path, os.getcwd()))
-        os.chdir(self.cwd)
```

Comment:
Tested and this change breaks himl.
In the above loop to discover all files it changed the cwd sequentially to the deepest level, this just reverts it back to the starting "root" dir.
If the working dir is not reverted to the "root" himl dir the execution fails.